apps/wallet/src/ui/app/pages/home/transfer-coin/PreviewTransfer.tsx

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useMemo } from 'react';
+
+import { Text } from '_app/shared/text';
+import { IconTooltip } from '_app/shared/tooltip';
+import { TxnAddress } from '_components/receipt-card/TxnAddress';
+import { TxnAmount } from '_components/receipt-card/TxnAmount';
+import { parseAmount } from '_helpers';
+import { useAppSelector, useFormatCoin, useCoinDecimals } from '_hooks';
+import { GAS_SYMBOL, GAS_TYPE_ARG } from '_redux/slices/sui-objects/Coin';
+
+export type TransferCoinFormProps = {
+    coinType: string;
+    gasCostEstimation: number | null;
+    onClearSubmitError: () => void;
```

Comment:
Is this necessary? I think we can remove it

Reply:
this is address here https://github.com/MystenLabs/sui/pull/8353/files#diff-a6809c89c32218f8f4af4b0211a48beb544e43bbb314a4f4b83db1bb53eafe24